src/core/linalg/src/sparse/4C_linalg_utils_sparse_algebra_math.hpp

```diff
@@ -135,6 +135,26 @@ namespace Core::LinAlg
    */
   std::shared_ptr<SparseMatrix> matrix_transpose(const SparseMatrix& A);
 
+  /**
+   * \brief Options for sparse matrix inverse
+   *
+   * A well-known disadvantage of incomplete factorizations is that the two factors can be unstable.
+   * This may occur, for instance, if matrix A is badly scaled, or if one of the pivotal elements
+   * occurs to be very small. In this case, a-priori diagonal perturbations may be effective.
+   *
+   * A_perturbed(i,j) = A(i,j)  i~=j
+   * A_perturbed(i,i) = alpha*sign(A(i,i))+rho*A(i,i)
+   *
+   */
+  struct OptionsSparseMatrixInverse
+  {
+    //! Absolute diagonal threshold
+    double alpha = 0.0;
+
+    //! Relative diagonal threshold
+    double rho = 1.0;
```

Comment:
1. Is "threshold" the right terminology? Isn't this rather like a "scaling factor"?
1. The documentation of the individual values is only useful in combination with the documentation of the `struct`. Just an observation. Might be fine.